Dockerfile

```diff
@@ -8,7 +11,7 @@ COPY main.go .
 COPY e2e_test.go .
 COPY pkg ./pkg
 RUN go test -race -v ./...
-RUN CGO_ENABLED=0 GOARCH=amd64 GOOS=linux go build -o /usr/local/bin/aws-ses-pop3-server
+RUN export GOARCH=$(echo $TARGETPLATFORM | cut -d/ -f2) ; CGO_ENABLED=0 GOOS=linux go build -o /usr/local/bin/aws-ses-pop3-server
```

Comment:
```suggestion
RUN GOARCH="$(cut -d/ -f2 <<< "${TARGETPLATFORM}")" CGO_ENABLED=0 GOOS=linux go build -o /usr/local/bin/aws-ses-pop3-server
```

Reply:
`<<<` is a bourne-again shell supported redirection, but we only have the bourne shell available in the container.

I've made the change to not use `export`, but left in the `echo <foo> | <bar>` construct so it'll still work.